### ISSUE TITLE
Label for the sign out link is not visible to screen readers and only visible on rollover

### ISSUE THREAD

**Issue by emma-sg** [emma-sg/roleypoly#3]
This is an issue because a) the logo on its own is unclear, and b) users accessing the site with assistive tools may not be able to understand what the sign out button is, and thus not be able to sign out.

The solution to this would simply be to make the label visible at all times, and within the `a` tag.
